targets/unity-v2/make.js

```diff
@@ -378,7 +418,11 @@ function getAuthParams(apiCall) {
     return "AuthType.None";
 }
 
-function getRequestActions(tabbing, apiCall) {
+function getInstanceParams(apiCall) {
+    return "authenticationContext, apiSettings"
+}
+
+function getRequestActions(tabbing, apiCall, isApiInstance = false) {
```

Comment:
Ok i see multiple things here that are wrong (we may actually fix some of them in C#, too). Please apply the following to EACH of the if (...) cases in this function:

1) If isApiInstance == false, then:
  a) Use values from request.AuthenticationContext (if it is not null), otherwise:
  b) Use values from PlayFabSettings.

2) If isApiInstance == true, then:
  a) Use values from request.AuthenticationContext (if it is not null), otherwise:
  b) Use values from this.AuthenticationContext (if it is not null), otherwise:
  c) Use values from PlayFabSettings

Using IsClientLoggedIn is ok if you want to have those functions, but please make sure you are using them right: instance method (this.IsClientLoggedIn()) should be called only when isApiInstance == true. <ClassName>.IsClientLoggedIn() must be called only when isApiInstance == false. Please make sure you are calling the right static method from the right class. If this template is meant to be used to generate each API class then please be careful to call specific static methods as you may end up calling it from the wrong API class.